README.md

```diff
@@ -285,7 +285,14 @@ same `channel_pat` will overwrite the old handler.
 Deregisters the event handler function that was previously registered via `on` or
 `register` method.
 
-
+## Logging
+You should define a logger in your app in order to make sure you get all the messages from the different components.<br>
+It will help you debug, and make sure you don't miss issues when they occur.<br>
```

Comment:
Do you need <br>?

Reply:
I want a new line:
![image](https://user-images.githubusercontent.com/7924802/89257097-3092fb00-d62e-11ea-8b3e-6254ba5a83ae.png)